docs/cloud/sources/googlecloudbilling.md

```diff
@@ -0,0 +1,63 @@
+# Event Source for Google Cloud Billing
+
+This event source receives messages from a [Google Cloud Billing][gc-billing] Budget by subscribing
+to a [Google Cloud Pub/Sub][gc-billing-events] topic.
+
+### Service Account
+
+A Service Account is required to authenticate the event source and allow it to interact with Google
+Cloud Billing budget.
+
+The service account must be granted an IAM Role with at least the following permissions:
+
+- `billing.budgets.get`
+- `billing.budgets.update`
+
+The following set of permissions is also required because this source delegates the management of the Pub/Sub subscription to the Pub/Sub Source.
```

Comment:
It's not 100% accurate because all sources are independent from one-another, so there is no "Pub/Sub source" involved.

Here is a suggestion or rewording:

```suggestion
The following set of permissions is also required to allow this source to manage the Pub/Sub topic and subscription:
```